### PR TITLE
Issue3

### PR DESCRIPTION
**Pull request by apurvabakshi**
Completed tasks as listed below:
1. List of images shown on homepage
2. Images are no wider than 400px and persist after server restart
3. Newest images appear first
4. HTTPS images are also getting accepted

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -23,6 +28,6 @@
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/js/bootstrap.min.js" integrity="sha384-alpBpkh1PFOepccYVYDB4do5UnbKysX5WZXm3XxPqe5iKTfUKjNkCk9SaVuEZflJ" crossorigin="anonymous"></script>
 
     <%= javascript_include_tag 'application' %>
-    <%= javascript_pack_tag 'application' %>
+    <%#= javascript_packrakes _tag 'application' %>
```
Comment (justinpearsonappfolio) [inline app/views/layouts/application.html.erb]: I think this might be a typo

**Comment by justinpearsonappfolio** [comment]
[Issue 3: Image Index](https://trello.com/c/tClWWxLs/4-issue-3-image-index)

**Comment by apurvabakshi** [comment]
[Issue 3: Image Index](https://trello.com/c/tClWWxLs/4-issue-3-image-index)

**Comment by apurvabakshi** [comment]
[Issue 3: Image Index](https://trello.com/c/tClWWxLs/4-issue-3-image-index)

**Comment by apurvabakshi** [comment]
[Issue 3: Image Index](https://trello.com/c/tClWWxLs/4-issue-3-image-index)

**Pull request merged**